lms/templates/wiki/history.html

```diff
@@ -216,15 +216,15 @@ <h2 id="preview-title">{% trans "Wiki Revision Preview" %}<span class="sr">, {%
         {% trans "Back to history view" %}
       </a>
       {% if article|can_write:user %}
-        <a href="#" class="btn btn-large btn-primary switch-to-revision">
+        <button type="button" class="btn btn-large btn-primary switch-to-revision">
           <span class="icon fa fa-flag" aria-hidden="true"></span>
           {% trans "Switch to this version" %}
-        </a>
+        </button>
       {% else %}
-        <a href="#" class="btn btn-large btn-primary disabled">
+        <button type="button" class="btn btn-large btn-primary disabled">
```

Comment:
Outside the scope. Here's [a new ticket](https://openedx.atlassian.net/browse/AC-565).